scss/main.scss

```diff
@@ -1,150 +1,92 @@
 @import 'scss/variables';
-@import 'node_modules/bootstrap-sass/assets/stylesheets/bootstrap';
+@import 'node_modules/bootstrap/scss/bootstrap';
 
+// flush footer to bottom
+html {
+    height: 100%;
+}
 
 body {
+    min-height: 100%;
     position: relative;
+    padding-bottom: 40px;
 }
 
-pre {
-    margin-bottom: 20px;
+footer {
+    // scss-lint:disable ImportantRule
+    position: absolute  !important;
+    right: 0;
+    bottom: 0;
+    left: 0;
```

Comment:
I think the fix for this is to remove padding-bottom on `body`?

Reply:
with the absolute positioning (in combination with `bottom: 0`) the footer is sticky (noticable if you search for hooks and the content height becomes lower than your browser height)

Reply:
Ah I see, that is better -- I didn't consider the hooks page